lib/modifiers/batch.js

```diff
@@ -0,0 +1,166 @@
+var through = require('through2');
+var _ = require('lodash');
+
+/**
+ * Returns a new stream that batches writes based on a specific interval, count of items, or both.
+ * All items read on the stream and will be re-emitted as part of an array of one or more items
+ * based on the criteria defined in the options.
+ *
+ * @method
+ * @static
+ * @since    TODO
+ * @category Util
+ *
+ * @param   {Object} [options]       - Define how items will be batched and written to the
+ *                                     the stream. Note that when both `time` and `count` are
+ *                                     defined, items will be written to the stream whenever
+ *                                     either one of the conditions is met.
+ * @param   {Number} [options.time]  - The interval, in milliseconds, to limit writes to.
+ *                                     During that time, all items read on the stream will
+ *                                     be collected and written as an array at the set
+ *                                     interval.
+ * @param   {Number} [options.count] - The number of items to buffer before writing all
+ *                                     of them in an array.
+ * @returns {Stream}                 - Transform stream.
+ *
+ * @example
+ * var input = through.obj();
+ *
+ * input.pipe(nodeStream.batch({ time: 100 }));
+ *
+ * input.write(1);
+ *
+ * setTimeout(function() {
+ *   input.write(2);
+ *   input.write(3);
+ * }, 100);
+ *
+ * setTimeout(function() {
+ *   input.write(4);
+ *   input.write(5);
+ * }, 200);
+ *
+ * // => [[1], [2, 3], [4, 5]]
+ * @example
```

Comment:
It works in the viewer, and it worked just now when I rendered your docs as well:

![image](https://cloud.githubusercontent.com/assets/2205537/21832299/1a3dd9f0-d778-11e6-86b9-80b3a3c475e9.png)

I like that is separates them into two blocks this way. Am I missing a problem? I can change it if you hate it, but it seems like it is working to me.